Mixpanel/Controller.cs

```diff
@@ -150,30 +173,16 @@ private IEnumerator SendData(MixpanelStorage.FlushType flushType)
             }
         }
 
-        private IEnumerator TrackIntegrationEvent()
+        private IEnumerator SendHttpEvent(string eventName, string apiToken, string distinctId, string properties)
         {
-            if (MixpanelStorage.HasIntegratedLibrary) {
-                yield break;
-            }
-            string body = "{\"event\":\"Integration\",\"properties\":{\"token\":\"85053bf24bba75239b16a601d9387e17\",\"mp_lib\":\"unity\",\"distinct_id\":\"" + MixpanelSettings.Instance.Token +"\"}}";
+            string body = "{\"event\":\"" + eventName + "\",\"properties\":{\"token\":\"" + 
+                        apiToken + "\",\"mp_lib\":\"unity\",\"$lib_version\":\""+ Mixpanel.MixpanelUnityVersion + "\",\"distinct_id\":\"" + distinctId + "\"" + properties + "}}";
             string payload = Convert.ToBase64String(Encoding.UTF8.GetBytes(body));
             WWWForm form = new WWWForm();
             form.AddField("data", payload);
               
             using (UnityWebRequest request = UnityWebRequest.Post(Config.TrackUrl, form)) {
                 yield return request.SendWebRequest();
-                #if UNITY_2020_1_OR_NEWER
-                if (request.result != UnityWebRequest.Result.Success)
-                #else
-                if (request.isHttpError || request.isNetworkError)
-                #endif
-                {
-                    Mixpanel.Log(request.error);
-                }
-                else
-                {
-                    MixpanelStorage.HasIntegratedLibrary = true;
```

Comment:
probably won't make much of a difference, but just noting that if we move this out of the result check, it will get set to true regardless of whether or not the track call was successful